meta-lxatac-bsp/recipes-bsp/barebox/files/patches/0102-ARM-stm32mp-add-Linux-Automation-TAC-board.patch

```diff
@@ -445,17 +408,14 @@ index 524d282a1db8..220d5b2c15e5 100644
  	select ARCH_STM32MP157
  	bool "Seeed Studio Odyssey"
 diff --git a/images/Makefile.stm32mp b/images/Makefile.stm32mp
-index cc70aee92388..80f6b9dfc8f5 100644
+index cb01eb2555e8..32a10f09e52d 100644
 --- a/images/Makefile.stm32mp
 +++ b/images/Makefile.stm32mp
-@@ -35,6 +35,10 @@ $(call build_stm32mp_image, CONFIG_MACH_STM32MP15X_EV1, start_stm32mp15x_ev1, st
+@@ -34,6 +34,7 @@ $(call build_stm32mp_image, CONFIG_MACH_STM32MP15XX_DKX, start_stm32mp15xx_dkx,
+ $(call build_stm32mp_image, CONFIG_MACH_STM32MP15X_EV1, start_stm32mp15x_ev1, stm32mp15x-ev1)
  
  $(call build_stm32mp_image, CONFIG_MACH_LXA_MC1, start_stm32mp157c_lxa_mc1, stm32mp157c-lxa-mc1)
++$(call build_stm32mp_image, CONFIG_MACH_LXA_TAC, start_stm32mp15xc_lxa_tac, stm32mp15xc-lxa-tac)
  
-+pblb-$(CONFIG_MACH_LXA_TAC) += start_stm32mp15xc_lxa_tac
```

Comment:
This reverts https://github.com/linux-automation/meta-lxatac/commit/e285dad4f06688c9df3f5b1d9720a7db5191542b which is likely not intended.

Reply:
It looks like the respective `fixup!` commit did not make it to the `v2024.03.0/customers/lxa/tac` branch or got force pushed away (sounds like something I would do by accident tbh).
I've made sure to push it to `v2024.03.0/customers/lxa/tac` again and have also included it in the last update of this PR.